.circleci/config.yml

```diff
@@ -37,6 +37,46 @@ commands:
             - run: sudo apt-get install ocl-icd-opencl-dev libhwloc-dev
       - run: git submodule sync
       - run: git submodule update --init
+  download-params:
+    steps:
+      - restore_cache:
+          name: Restore parameters cache
+          keys:
+            - 'v25-2k-lotus-params'
+          paths:
+            - /var/tmp/filecoin-proof-parameters/
+      - run:  lotus fetch-params 2048
+      - save_cache:
+          name: Save parameters cache
+          key: 'v25-2k-lotus-params'
+          paths:
+            - /var/tmp/filecoin-proof-parameters/
+  lotus-artefacts:
```

Comment:
This part could use a bit of love. We do load /tmp/lotus-artefacts , but we are also running all the docker part afterwards, which is redundant.

Reply:
Fair enough but I think this is a great start, so happy to do that later